packages/commonwealth/server/migrations/20240510175021-create-profile-tags.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction(async (t) => {
+      await queryInterface.createTable(
+        'ProfileTags',
+        {
+          id: {
```

Comment:
Same issue here for natural vs surrogate PK. Also this has the advantage of data integrity. In this case a Profile can link the same tag multiple times which could have unintended consequences and tech debt when displaying this data on the frontend.

Same comment for the community tags migration

Reply:
Done [here](https://github.com/hicommonwealth/commonwealth/pull/7824/commits/9f90edf2c69a55ced91dc24f8bc6bc435b99b938)